locale/Makefile

```diff
@@ -50,14 +50,16 @@ tx-pull: $(EDITFILES)
 
 tx-update: tx-pull
 	@echo
-	@echo Run rake plugin:gettext[$(DOMAIN)] from the Foreman installation, then make -C locale mo-files to finish
+	@echo Run rake plugin:gettext[$(DOMAIN)] from the Foreman installation. \
+		If this is for a release, run make -C locale mo-files from the Katello directory to finish. \
+		.mo files should not be uploaded to GitHub, they should only be packed into the gem.
 	@echo
 
 mo-files: $(MOFILES)
-	git add $(POFILES) $(POTFILE) ../locale/*/LC_MESSAGES
-	git commit -m "i18n - pulling from tx"
+	git add -f $(POFILES) $(POTFILE) ../locale/*/LC_MESSAGES
+	git commit -m "i18n - creating .mo files"
 	@echo
-	@echo Changes commited!
+	@echo Changes commited! Do not push this commit to GitHub as it contains .mo files.
```

Comment:
Any reason we can't just add `.mo` to `.gitignore`?

Reply:
They are indeed ignored

Reply:
In that case, is this warning needed? I guess you'd have a blank commit then, okay :)